sonar-application/src/main/java/org/sonar/application/ClusterProperties.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.application;
+
+import org.sonar.process.Props;
+
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Properties of the cluster configuration
+ */
+public final class ClusterProperties {
```

Comment:
I mimic the other XXProperties class (ProcessProperties, previous ClusterProperties, etc).
The best validation is done by Hazelcast when creating the instance, if something is wrong an Hazelcast exception is thrown (making App failing fast).